spec/models/article_spec.rb

```diff
@@ -137,3 +137,28 @@
     end
   end
 end
+
```

Comment:
All these monthly specs need specs to cover months that don't have 30 or 31 days, as well as the leap year.